update.sh

```diff
@@ -216,10 +219,31 @@ jobs:
   build:
     name: ${version} on ${variant}
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix: \${{fromJson('{\"arch\":${arch_json}}')}}
     steps:
-      - uses: actions/checkout@v2
-      - run: sudo apt-get install bats
-      - run: ./test-build.sh ${version} ${variant}" > ".github/workflows/${version}-${variant}.yml"
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install bats
+        run: sudo apt-get install bats
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Is it worth skipping these steps for amd64? EX: `if: matrix.arch != 'amd64'`

Reply:
Skipping it for amd64 would work just fine. The step usually takes around 5 seconds anyway.

Reply:
Yeah, I wasn't sure which of the other step below were needed for the non-QEMU arch builds